app/code/Magento/Sales/etc/db_schema.xml

```diff
@@ -876,6 +876,7 @@
         <column xsi:type="int" name="order_id" padding="10" unsigned="true" nullable="false" identity="false"
                 comment="Order Id"/>
         <column xsi:type="text" name="track_number" nullable="true" comment="Number"/>
+        <column xsi:type="varchar" name="track_url" nullable="true" length="255" comment="Url"/>
```

Comment:
Implementing this feature as an extension attribute would allow to avoid schema modification for the existing table.